### PR TITLE
[zh]sync-pod-priority-preemption.md

### PR DESCRIPTION
**Pull request by asa3311**
Related to task-1 of issue https://github.com/kubernetes/website/issues/39985.

content/en/docs/concepts/scheduling-eviction/pod-priority-preemption.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 1]
```diff
@@ -123,7 +124,8 @@ The `description` field is an arbitrary string. It is meant to tell users of the
 cluster when they should use this PriorityClass.
 -->
 PriorityClass 对象可以设置任何小于或等于 10 亿的 32 位整数值。
-较大的数字是为通常不应被抢占或驱逐的关键的系统 Pod 所保留的。
+这意味着 PriorityClass 对象的值范围是 从 -2147483648 到 1000000000（含）。
```
Comment (howieyuen) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md:127]:
```suggestion
这意味着 PriorityClass 对象的值范围是从 -2,147,483,648 到 1,000,000,000（含）。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 2]
```diff
@@ -123,7 +124,8 @@ The `description` field is an arbitrary string. It is meant to tell users of the
...
+较大的数字保留用于代表关键系统 Pod 的内置 PriorityClasses。
```
Comment (howieyuen) [inline content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md:128]:
```suggestion
保留更大的数字，用于表示关键系统 Pod 的内置 PriorityClass。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md (hunk 1 above)
Comment (asa3311) [reply content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md]: thanks，done。

File: content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md (hunk 2 above)
Comment (asa3311) [reply content/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption.md]: thanks，done。

**Review by howieyuen — COMMENTED** [review]
many thanks for your work, 🌹 
/lgtm
/approve

**Pull request merged**